README.md

```diff
@@ -50,7 +53,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:residents].delete("Belle")
+town[:castle, :guests] << "Belle"
```

Comment:
You need to separate castle and guests like so since they are distinct objects: `town[:castle][:guests]`